app/lib/dfe/bigquery/application_metrics.rb

```diff
@@ -106,6 +122,46 @@ def self.sex_query(cycle_week:)
         .to_sql
       end
 
+      def self.route_into_teaching_query(cycle_week:)
+        where(
+          recruitment_cycle_year:,
+          cycle_week:,
+          subject_filter_category: 'Total excluding Further Education',
+          nonsubject_filter_category: 'Route into teaching',
+        )
```

Comment:
I really wish to use scopes but my simple ORM (😂) can't handle and it will be too much complexity without too much of benefit.